### ISSUE TITLE
Why does the loss always be 0 when fine-tuning llava-onevision?

### ISSUE THREAD

**Issue by 98986oiuoy** [zjysteven/lmms-finetune#45]
I directly used the data and parameters of the example for picture fine-tuning, and the loss and grad_norm values in the echo of the terminal are always 0, and adjusting the learning rate multiple times has no effect.

![76e24ef7b6933bd1182c294937dac8f](https://github.com/user-attachments/assets/d63ddba7-e0e3-47b8-913b-78f6ff40cbe6)

**Comment by 98986oiuoy** [comment]
Resolved MODEL_MAX_LEN adjusted to 2048

**Comment by zjysteven** [comment]
In case anyone else has the same problem, please see a detailed explanation of this in #43

**Comment by Kshitijpawar** [comment]
> Resolved MODEL_MAX_LEN adjusted to 2048 

@98986oiuoy 
How did you loss look like, for me it fluctuates a lot, but at the end of 5 epochs its not near to the one in the github notebook
<img width="1384" alt="Screenshot 2024-11-27 at 1 52 32 PM" src="https://github.com/user-attachments/assets/884bf8a2-d9b7-4950-b9ca-ea1d29d30886">
